app/views/site_channels/_nav.html.slim

```diff
@@ -0,0 +1,12 @@
+nav.navbar.navbar-default.navbar-static-top.top-nav-collapse
+  .container-fluid
+    .menu-container
+      .nav.pull-right
+        = link_to(I18n.t("shared.cancel"), home_publishers_path, class: 'btn btn-secondary btn-highlight btn-cancel pull-right')
+
+      .nav.pull-left
+        / = link_to(root_path) do
+        .navbar-brand.page-title= t "publishers.add_channel"
```

Comment:
minor: if we ever add non-"Add channel" related actions to SiteChannelsController this would show as well.

it is all right for now